src/test/java/timetables_sort_use_case/application_business/TimetablesSortInteractorTest.java

```diff
@@ -0,0 +1,159 @@
+package timetables_sort_use_case.application_business;
+
+import entities.*;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import timetables_sort_use_case.interface_adapters.TestTimetablesSortPresenter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+class TimetablesSortInteractorTest {
+    private TimetablesSortInteractor interactor;
+    private TestTimetablesSortPresenter presenter;
+
+    @BeforeEach
+    void setUp() {
+        presenter = new TestTimetablesSortPresenter();
```

Comment:
A line or two of what's happening here could help clarify the purpose of this daunting code block. 
(Even if it's just "prep initialization of some dummy values", I think).

Reply:
done!